umake/frameworks/ide.py

```diff
@@ -477,6 +477,27 @@ def __init__(self, category):
                          icon_filename='product.png')
 
 
+class Rider(BaseJetBrains):
+    """The JetBrains  cross-platform .NET IDE"""
+    download_keyword = 'RD'
+    executable = "rider.sh"
+
+    def __init__(self, category):
+        super().__init__(name="Rider",
+                         description=_("The JetBrains  cross-platform .NET IDE"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         packages_requirements=['mono-devel'],
+                         dir_to_decompress_in_tarball='riderRS-*',
```

Comment:
The folder is actually "Rider-"
Probably they changed it to reflect what the other non eap releases are